src/Workspaces/Core/Portable/Diagnostics/InternalDiagnosticsOptions.cs

```diff
@@ -8,9 +8,6 @@ internal static class InternalDiagnosticsOptions
     {
         private const string LocalRegistryPath = @"Roslyn\Internal\Diagnostics\";
 
-        public static readonly Option<bool> BlueSquiggleForBuildDiagnostic = new Option<bool>(nameof(InternalDiagnosticsOptions), "Blue Squiggle For Build Diagnostic", defaultValue: false,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + "Blue Squiggle For Build Diagnostic"));
```

Comment:
pure question: why was this removed?  I have no problem with it going.  I'm just trying to understand the reasoning/discussion around it.

Reply:
It's just not needed anymore.

---
In reply to: [287436212](https://github.com/dotnet/roslyn/pull/35902#discussion_r287436212) [](ancestors = 287436212)

Reply:
it was added to test user experience long time ago and to test deduplication and no longer used. and we made decision not to show different color for build errors.

Reply:
Thanks for the explanation.  It's very appreciated!